File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>(k);

        for(int n : nums){
            pq.offer(n);
            if(pq.size()>k){
                pq.poll();
            }
        }
        return pq.peek();
    }
}

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.py
class Solution(object):
    def findKthLargest(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        min_heap = []

        for i in nums:
            heapq.heappush(min_heap, i)

            if len(min_heap) > k:
                heapq.heappop(min_heap)

        return min_heap[0]
        